### PR TITLE
Initializing core files

### PR DESCRIPTION
**Pull request by jeradrutnam**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "ballerina-extension",
+    "title": "Ballerina Brackets Extension",
+    "description": "File Format Support for WSO2 Ballerina Development Environment (http://ballerinalang.org/)",
```
Comment (dakshika) [inline package.json]: Syntax highlighting

File: package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "ballerina-extension",
+    "title": "Ballerina Brackets Extension",
```
Comment (dakshika) [inline package.json]: Isn't "ballerina" enough?

Comment (jeradrutnam) [inline package.json]: Yes, we can change it to "Ballerina"

File: package.json (hunk 1 above)
Comment (jeradrutnam) [inline package.json]: How about "Syntax Highlighting Support for WSO2 Ballerina (http://ballerinalang.org/)" ?

Comment (dakshika) [inline package.json]: +1

**Pull request merged**